build/docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM ${GOLANG_IMAGE} as builder
 
 COPY /build/swi-k8s-opentelemetry-collector.yaml /src/swi-k8s-opentelemetry-collector.yaml
 
-RUN GO111MODULE=on go install go.opentelemetry.io/collector/cmd/builder@v0.64.0
+RUN GO111MODULE=on go install go.opentelemetry.io/collector/cmd/builder@v0.67.0
```

Comment:
You should use the builder version aligned with processors, so please upgrade this to `0.68.0`. Build notifies you about that: https://github.com/solarwinds/swi-k8s-opentelemetry-collector/actions/runs/3845213893/jobs/6549098289#step:5:383

Reply:
I tired, but there's no build with version 0.68.0 :/